plugins/modules/ndo_mcp_global_policy.py

```diff
@@ -0,0 +1,336 @@
+    aliases: [ mcp_global_policy_uuid ]
+  description:
+    description:
+    - The description of the MCP Global Policy.
+    - Providing an empty string will remove the O(description="") from the MCP Global Policy.
+    type: str
+  admin_state:
+    description:
+    - The administrative state of the MCP Global Policy.
+    - Defaults to C(enabled) when unset during creation.
+    type: str
+    choices: [ enabled, disabled ]
+  key:
+    description:
+    - The key of the MCP Global Policy.
+    type: str
+  per_vlan:
+    description:
+    - Enable or disable MCP packets being sent to each End Point Group (EPG).
+    - Defaults to C(disabled) when unset during creation.
+    type: str
+    choices: [ enabled, disabled ]
+    aliases: [ per_epg, mcp_pdu_per_vlan ]
+  loop_detection_factor:
+    description:
+    - The amount of MCP packets that will be received before port disable loop protection action takes place.
+    - Defaults to 3 when unset during creation.
+    - The value must be between 0 and 255.
+    type: int
+    aliases: [ loop_factor, loop_detection_mult_factor ]
+  port_disable:
+    description:
+    - Enable or disable port disabling when MCP packets are received.
+    - Defaults to C(enabled) when unset during creation.
+    type: str
+    choices: [ enabled, disabled ]
+    aliases: [ port_disable_protection ]
+  initial_delay_time:
+    description:
+    - The MCP initial delay time in seconds.
+    - Defaults to 180 when unset during creation.
+    - The value must be between 0 and 1800.
+    type: int
+    aliases: [ initial_delay ]
+  transmission_frequency_sec:
+    description:
+    - The MCP transmission frequency in seconds.
+    - Defaults to 2 when unset during creation.
+    - The value must be between 0 and 300.
+    type: int
+    aliases: [ tx_freq ]
+  transmission_frequency_msec:
+    description:
+    - The MCP transmission frequency in milliseconds.
+    - Defaults to 0 when unset during creation.
+    - The value must be between 0 and 999.
+    type: int
+    aliases: [ tx_freq_ms ]
+  state:
+    description:
+    - Use C(absent) for removing.
+    - Use C(query) for listing an object or multiple objects.
+    - Use C(present) for creating or updating.
+    type: str
+    choices: [ absent, query, present ]
+    default: query
+notes:
+- The O(template) must exist before using this module in your playbook.
+  Use M(cisco.mso.ndo_template) to create the Fabric Policy template.
+- Attempts to create any additional MCP Global Policies will only update the existing
+  object in the Fabric Policy template.
+seealso:
+- module: cisco.mso.ndo_template
+extends_documentation_fragment: cisco.mso.modules
+"""
+
+EXAMPLES = r"""
+- name: Create the MCP Global Policy object
+  cisco.mso.ndo_mcp_global_policy:
+    host: mso_host
+    username: admin
+    password: SomeSecretPassword
+    template: fabric_template
+    name: mcp_global_policy_1
+    key: cisco
+    state: present
+  register: mcp_global_policy_1
```

Comment:
do we want the register to be the same value as the example with all attributes?

Reply:
Changed